app/lib/features/tasks/sheets/create_update_task_item.dart

```diff
@@ -0,0 +1,228 @@
+          key: _formKey,
+          child: Column(
+            mainAxisSize: MainAxisSize.min,
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: [
+              const SizedBox(height: 20),
+              const Divider(
+                indent: 150,
+                endIndent: 150,
+                thickness: 2,
+              ),
+              const SizedBox(height: 20),
+              Text(
+                L10n.of(context).addTask,
+                textAlign: TextAlign.center,
+                style: Theme.of(context).textTheme.titleMedium,
+              ),
+              const SizedBox(height: 20),
+              _widgetTaskName(),
+              const SizedBox(height: 20),
+              _widgetDescriptionName(),
+              const SizedBox(height: 20),
+              _widgetDueDate(),
+              const SizedBox(height: 20),
+              _widgetAddButton(),
+              const SizedBox(height: 20),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _widgetTaskName() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          L10n.of(context).taskName,
+          style: Theme.of(context).textTheme.bodySmall,
+        ),
+        const SizedBox(height: 5),
+        TextFormField(
+          autofocus: true,
+          decoration: InputDecoration(
+            hintText: L10n.of(context).name,
+          ),
+          autovalidateMode: AutovalidateMode.onUserInteraction,
+          controller: _taskNameController,
+          validator: (value) => (value?.isNotEmpty == true)
+              ? null
+              : L10n.of(context).pleaseEnterAName,
+        ),
+      ],
+    );
+  }
+
+  Widget _widgetDescriptionName() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          L10n.of(context).description,
+          style: Theme.of(context).textTheme.bodySmall,
+        ),
+        const SizedBox(height: 5),
+        TextFormField(
+          decoration: InputDecoration(
+            hintText: L10n.of(context).description,
+          ),
+          minLines: 4,
+          maxLines: 4,
+          controller: _taskDescriptionController,
+        ),
+      ],
+    );
+  }
+
+  Widget _widgetDueDate() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          L10n.of(context).dueDate,
+          style: Theme.of(context).textTheme.bodySmall,
+        ),
+        const SizedBox(height: 5),
+        TextFormField(
+          readOnly: true,
+          decoration: InputDecoration(
+            hintText: L10n.of(context).dueDate,
+            suffixIcon: IconButton(
+              onPressed: selectDueDate,
+              icon: const Icon(Icons.calendar_month),
+            ),
+          ),
+          onTap: selectDueDate,
+          controller: _taskDueDateController,
+        ),
+      ],
```

Comment:
when changing the tasks due date the most common way to think about it that something is due "today" or "tomorrow" (and patterns alike). That's why for changing the due-date we have a custom date picker that allows for those to easily be picked. I understand the want to not reuse that (rather ugly) widget but we need to keep that use case in mind. Maybe by adding two `ActerInlineTextButtons` for quick setting? (the following is not real applicable code, just to translate an idea)
```suggestion
        TextFormField(
          readOnly: true,
          decoration: InputDecoration(
            hintText: L10n.of(context).dueDate,
            suffixIcon: IconButton(
              onPressed: selectDueDate,
              icon: const Icon(Icons.calendar_month),
            ),
          ),
          onTap: selectDueDate,
          controller: _taskDueDateController,
        ),
        Row(children:[
          ActerInlineTextButton(child: Text(L10n.of(context).today, onPressed: () => setState(() => { selectedDate = DateTime.today() }), 
           ActerInlineTextButton(child: Text(L10n.of(context).tomorrow, onPressed: () => setState(() => { selectedDate = DateTime.tomorrow() }),
           ])
      ],
```

Reply:
Added options as below.

![Screenshot 2024-06-05 at 6 35 29 PM](https://github.com/acterglobal/a3/assets/51526480/09902106-a5ed-4e83-be07-46cae8bb590f)